checker/src/org/checkerframework/checker/units/UnitsAnnotatedTypeFactory.java

```diff
@@ -95,6 +94,9 @@ public AnnotationMirror aliasedAnnotation(AnnotationMirror a) {
     @Override
     @SuppressWarnings("unchecked")
     protected Set<Class<? extends Annotation>> createSupportedTypeQualifiers() {
+        // make sure we have an empty set of units relations at the start of this method
```

Comment:
I had it there at one point because unitsRel set wasn't instantiated, turns out it is removable now.